akka-cluster-tools/src/test/scala/akka/cluster/pubsub/DistributedPubSubMediatorRouterSpec.scala

```diff
@@ -25,6 +25,8 @@ object DistributedPubSubMediatorRouterSpec {
 trait DistributedPubSubMediatorRouterSpec { this: WordSpecLike with TestKit with ImplicitSender ⇒
   def nonUnwrappingPubSub(mediator: ActorRef, testActor: ActorRef, msg: Any) {
 
+    system.eventStream.subscribe(testActor, classOf[DeadLetter])
```

Comment:
I think this should be installed in the specific test that is using this, there is a risk that other tests also publish stuff to deadletters. I also think you should use a separate `TestProbe` for this instead of `testActor` to not interfere with other usages of `testActor`